tests/helper/helper_dev.go

```diff
@@ -77,22 +88,33 @@ import (
 	To run independent tests, previous formats should be used instead.
 
 	It("should do ... in dev mode", func() {
-		helper.RunDevMode(func(session *gexec.Session) {
+		helper.RunDevMode(func(session *gexec.Session, out []byte, err []byte) {
 			// test on dev mode
+			// out and err are contents of std/err output when dev mode is started
 		})
 	})
+
+	# Waiting for file synchronisation to finish
+
+	The method session.WaitSync() can be used to wait for the synchronization of files to finish.
+	The method returns the contents of std/err output since the end of the dev mode started, and until the end of the synchronization.
 */
 type DevSession struct {
 	session *gexec.Session
 }
 
 // StartDevMode starts a dev session with `odo dev`
-func StartDevMode() DevSession {
+func StartDevMode() (DevSession, []byte, []byte) {
```

Comment:
Can you update the comment to include info about the two `[]byte`s we return here?